jupyros/ros_widgets.py

```diff
@@ -325,4 +344,112 @@ def call_srv(arg):
     widget_list.append(call_btn)
     vbox = widgets.VBox(children=widget_list)
 
-    return vbox
\ No newline at end of file
+    return vbox
+
+
+class TurtleWidget:
+    def __init__(self, width=1600, height=1600, turtle_size=100):
+        self.turtles = {}
+        self.turtle_size = turtle_size
+        self.canvas_middle = {"x": width // 2,
+                              "y": height // 2,
+                              "theta": 0}
+
+        # Three layers for the canvas: 0-background, 1-paths, 2-turtles
+        self.canvas = ipycanvas.MultiCanvas(3,
+                                            width=width, height=height,
+                                            layout={"width": "100%"})
+
+        # Water background
+        self.canvas[0].fill_style = '#4556FF'
```

Comment:
should this be configurable? Could be an argument with default value to the contstructor.